src/components/Heading/Heading.tsx

```diff
@@ -17,21 +17,44 @@ export const VARIANTS = [
    */ 'info',
 ] as const;
 export type Variant = (typeof VARIANTS)[number];
-export type HeadingElement = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+export type HeadingElement =
+  | 'h1'
+  | 'h2'
+  | 'h3'
+  | 'h4'
+  | 'h5'
+  | /** @deprecated */ 'h6';
```

Comment:
:o are <h6> not recommended anymore? I don't think I've ever actually used an <h6> just wondering where this is coming from?